osu.Game/Rulesets/Edit/Checks/CheckMutedObjects.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Utils;
+using osu.Game.Rulesets.Edit.Checks.Components;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+
+namespace osu.Game.Rulesets.Edit.Checks
+{
+    public class CheckMutedObjects : ICheck
+    {
+        /// <summary>
+        /// Volume percentages lower than this are typically inaudible.
+        /// </summary>
+        private const int muted_threshold = 5;
```

Comment:
Nit: xmldoc is a touch misleading, given that the conditionals that use this do not use strict equality (so this should read "lower than or equal to" for 100% correctness).